microsoft-edge/extensions-chromium/publish/api/using-addons-api.md

```diff
@@ -20,7 +20,8 @@ To submit suggestions and feedback, enter an [Issue about the Add-ons API](https
 
 | Language | Package | Source code | Notes |
 | --- | --- | --- | --- |
-| Node.js | [Edge Webstore Upload](https://www.npmjs.com/package/@plasmo-corp/ewu) | [GitHub](https://github.com/plasmo-corp/edge-webstore-upload)| This is a third party library. Microsoft makes no warranties, express or implied, with respect to the information provided here. |
+| Node.js | [Microsoft Edge Addons API for NodeJS](https://www.npmjs.com/package/@plasmo-corp/ewu) | [PlasmoHQ / edge-addons-api](https://github.com/PlasmoHQ/edge-addons-api) repo | This is a third-party library.  Microsoft makes no warranties, express or implied, with respect to the information provided here. |
+| .NET / C# | [Microsoft Edge Addons API in .NET](https://www.nuget.org/packages/SantoshPisini.EdgeAddonsAPI) | [SantoshPisini / edge-addons-api](https://github.com/SantoshPisini/edge-addons-api) repo | This is a third-party library.  Microsoft makes no warranties, express or implied, with respect to the information provided here. |
```

Comment:
* Decide whether to commit to maintaining the library long-term.
* If so, copy|move directories from /SantoshPisini/-owned repo (https://github.com/SantoshPisini/edge-addons-api) to /microsoft/-owned repo (https://github.com/microsoft/MicrosoftEdge-Extensions).

@sivMSFT, @SantoshPisini, @dhruv16dhr, Edge Extensions Engineering team